src/ontology/uberon-edit.obo

```diff
@@ -222485,6 +222485,16 @@ relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500106
+name: hormion
+def: "Craniometric point along the midline of the sphenovomerine suture on the ventral (inferior in humans) side." [https://locatelambda.org/manual, ISBN:9780873659505]
+synonym: "vomerine notch" EXACT []
```

Comment:
Do you have a dbxref to support this being an exact synonym?
I see it mentioned [here](https://palaeo-electronica.org/content/2019/2417-narial-morphology), and I'm not sure if it's the same. If it is indeed the same, it says it is only found in mammals, so a taxon restriction may be appropriate and they provide another synonym: incisura vomeris (Latin).

Reply:
you're right, I'll remove it.